addons/website/tests/test_menu.py

```diff
@@ -2,9 +2,10 @@
 
 import json
 
-from odoo.tests import common
+from odoo.tests import common, tagged
 
 
+@tagged('test_menu')
```

Comment:
I didn't see anything in commit messages about those new tags usage. What are they used for and why are they needed?
Should we also add tag now in new test classes?

Reply:
It's just to allow launching them easily using ``--test-enable --test-tags test_menu``

Reply:
As @Feyensv mentioned, we can already test a class without a specific tag, it's why I was asking. I'd not add those tags